build/proxyAgent.mjs

```diff
@@ -0,0 +1,30 @@
+import HttpsProxyAgent from 'https-proxy-agent';
+
+export default () => {
```

Comment:
i'm seeing some linting issues on this one, make sure you run `npm run lint:fix`